extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -84,12 +88,12 @@
 
     public static final String FREE_TEXT_QUERY_TERM_LIMIT = "sc.freetext.querytermlimit";
 
-    public static final String FREE_TEXT_DOC_TABLENAME = "sc.freetext.doctable";
-    public static final String FREE_TEXT_TERM_TABLENAME = "sc.freetext.termtable";
-    public static final String GEO_TABLENAME = "sc.geo.table";
+//    public static final String FREE_TEXT_DOC_TABLENAME = "sc.freetext.doctable";
```

Comment:
These zombies  are here to remind me to ask you reviewers if you think I should deprecate or delete?  I deleted a couple public methods that might be deprecated also.  I think they should be deleted, since we can't preserve the correct behaivior.